tdrs-backend/tdpservice/parsers/test/factories.py

```diff
@@ -15,7 +70,7 @@ class Meta:
     column_number = 1
     item_number = 1
     field_name = "test field name"
-    category = 1
+    category = "1"
```

Comment:
TODO: update to use new `choices`

Reply:
Hmm, after playing with the new `ParserErrorCategoryChoices`, they were left flexible such that these still work. It looks more convoluted to try using it formally. Let me know what you think on which direction to take this.

Reply:
I like the new choices model, it si more readable